clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Button, Divider, Stack } from "@fidesui/react";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+
+import { CloseSolidIcon } from "~/features/common/Icon";
+
+import Stepper from "../common/Stepper";
+import steps from "./constants";
+import OrganizationInfoForm from "./OrganizationInfoForm";
+
+const ConfigWizardWalkthrough = () => {
+  const router = useRouter();
+  const [step, setStep] = useState<number>(1);
+
+  const handleCancelSetup = () => {
+    // Save progress
```

Comment:
does this save progress? or is this a todo?

Reply:
It's a to do - I'm going to address handling cancelling across the wizard in a separate PR, but I left this comment here as a reminder to myself.